instances/widgets.treasury-factory.near/widget/components/AccountInput.jsx

```diff
@@ -111,9 +126,13 @@ return (
         },
       }}
     />
-    {account && !isValidAccount && (
+    {account && (
       <div style={{ color: "red" }} className="text-sm mt-1">
-        Please enter valid account ID
+        {hasLockup
```

Comment:
this will create issue for other places where we use `AccountInput` and don't need to check for lockup, we should send props to make sure we only check if needed
After change, please manually check once, the other places we use `AccountInput` that it works as expected